src/main/java/org/cactoos/map/Merged.java

```diff
@@ -42,11 +42,12 @@
      */
     @SafeVarargs
     public Merged(final Map<K, V>... maps) {
-        super(() -> new MapOf<>(
-            new ListOf<>(maps)
-                .stream()
-                .flatMap(map -> map.entrySet().stream())
-                .collect(Collectors.toList())
+        super(
+            new MapOf<>(
+                new ListOf<>(maps)
+                    .stream()
+                    .flatMap(map -> map.entrySet().stream())
+                    .collect(Collectors.toList())
```

Comment:
@andreoss can't we manage to avoid creating an intermediate list just to iterate over it in MapOf? I'm not very clear about what streams provide us for that though...

Reply:
@andreoss well actually, this is out of scope of this PR, let's forget about it, I will create an issue.